### ISSUE TITLE
middleware/federation: config error

### ISSUE THREAD

**Issue by chrisohaver** [coredns/coredns#1024]
I'm seeing the following error at start up when using federation ...

```
CoreDNS-010
linux/amd64, go1.8.1, 9452a0a3-dirty
2017/09/06 17:10:19 middleware/federation: at least one name to zone federation expected
```

for the following configuration ...

```
    .:53 {
      health
      errors
      log
      autopath cluster.local @kubernetes
      kubernetes cluster.local {
        pods verified
      }
      federation cluster.local {
        prod prod.feddomain.com
      }
    }
```

**Comment by fturib** [comment]
@miekg : I think would he easier if we can use the 'assignee' as a flag to know if someone is working on an issue and, if yes, who.
If you agree, that would means : can you assign yourself to the issues you are working on ? 
and on the other hand avoid to work on issues already assigned to someone (unless agreement in the issue conversation).

By the way do you expect those 2 issues (#1024, #1025) being fixed for 1.0.0. ?
Note that once fixed we may found other issue down the road in manual tests.

**Comment by miekg** [comment]
> @miekg : I think would he easier if we can use the 'assignee' as a flag to know if someone is working on > an issue and, if yes, who. If you agree, that would means : can you assign yourself to the issues you are working on ?

I dislike the assignee button a bit, because it is so binary and there is no other feedback than that someone is assigned... I sometimes work on an issue for a bit, fail and then don't come back to it after a day. Should I keep the issue assigned to me all the time? Maybe assign and a comment on a issue?

In this particular case: I touched this code last; so that puts me in a position to fix stuff....

> By the way do you expect those 2 issues (#1024, #1025) being fixed for 1.0.0. ?

well, yes :)

> Note that once fixed we may found other issue down the road in manual tests.

Yep. I also don't know how to have some good unit tests for these, let alone integration tests that actually exercise these code paths... :(
